docs/install-1.7.md

```diff
@@ -21,65 +30,75 @@ kubernetes infrastructure CAs or certificates. This script should be
 `source`ed to define all of the variables it contains in the current
 shell process.
 
-The aggregator is a new feature of kubernetes and is an alpha API
-running as a separate pod in kubernetes v1.6. The aggregator is
-enabled by default in kubernetes v1.7 as a beta API, and is the
-default secure endpoint directly integrated into the core kubernetes
-APIServer.
+For background on why we need to deal with certificates and CA's at all, you 
+may read the [the auth doc](auth.md) (this is not required, however).
+
+## Install `cfssl` Tools
+
+Before we continue, you'll need to install the `cfssl` tools:
+
+```console
+go get -u github.com/cloudflare/cfssl/cmd/...
+```
+
+## Create a Certificates Directory
+
+Please create a fresh directory called `certs` to hold the configuration
+and certificates that we'll generate in the following steps:
 
-For background on why we're messing with certificates and CA's at all,
-please check [the auth doc](auth.md).
+```console
+mkdir certs
+```
```

Comment:
do we want to change dir to there now?  The original steps below don't cd and copy certs to the current directory (root of the SC source for me)

Reply:
Probably, but I haven't fully tested these instructions (I forked them and made changes there). I'd just like to get them into the right file for now and then improve in follow-ups. Cool with you?

Reply:
I ended up addressing this after all